pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -127,6 +130,8 @@ func NewAttachDetachController(
 		cloud:       cloud,
 	}
 
+	adc.flexVolumeProber = flexvolume.NewFlexVolumeProber(config.FlexVolumePluginDir)
```

Comment:
See https://github.com/kubernetes/kubernetes/pull/49668 for the implementation one layer up.